skate-plugin/src/main/kotlin/com/slack/sgp/intellij/ui/FilesListPanel.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij.ui
+
+import com.intellij.CommonBundle
+import com.intellij.openapi.fileChooser.FileChooser
+import com.intellij.openapi.fileChooser.FileChooserDescriptor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.NlsContexts.DialogTitle
+import com.intellij.openapi.util.NlsContexts.Label
+import com.intellij.openapi.vcs.changes.ui.VirtualFileListCellRenderer
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.ui.ToolbarDecorator
+import com.intellij.ui.components.JBList
+import javax.swing.DefaultListModel
+import javax.swing.JPanel
+import javax.swing.ListSelectionModel
+
+@Suppress("UnstableApiUsage") // For NlsContexts annotations
```

Comment:
Let's remove this, you can suppress it in your IDE settings instead

Reply:
7e8b41f